release-1.8/release_notes_draft.md

```diff
@@ -91,6 +91,17 @@ fundamental aspect of a secure cluster.
   to `true` or left blank, you won't see any change in behavior. If it's set to
   `false`, previously dormant StatefulSets might become active after upgrading.
 
+* CronJobs has been promoted to `v1beta1` which is now turned on by default.
+  Although version `v2alpah1` is still available, it is deprecated.  Migrate to
+  `batch/v1beta1.CronJobs`.  Additionally, if you are running cluster in
+  high-availability configuration this might cause some troubles when upgrading
+  clusters.  The reason for that is that, when in the middle of a rolling upgrade
+  the new controllers will be getting not found errors, because they rely on the
+  new version of the resource.
```

Comment:
Additionally, upgrading cluster in high-availability configuration might return errors.  The new controllers rely on the latest version of the resources. If the expected version is not found during rolling upgrade, the system throws resource not found errors.

I tried to cut down the long sentence. Feel free to rewrite if my suggestion has changed the meaning.

Reply:
That's perfect. Updated once again. ptal